### ISSUE TITLE
TimeZone is only collected locally

### ISSUE THREAD

**Issue by dpaulson45** [dpaulson45/HealthChecker#174]
Currently, the TimeZone is running within the local PowerShell session. So if we are collecting something from a remote server and they are in different time zones and it wouldn't be reporting correctly. 

This might not be an issue later one when we get the script working for multi remote execution.

**Comment by dpaulson45** [comment]
Make sure we use ([System.TimeZoneInfo]::Local).StandardName when addressing this.

**Comment by dpaulson45** [comment]
v2.34 is addressed in commit 0cc2fb22b2983a748bb28f46c3ca5cd8408a2f7b

**Comment by dpaulson45** [comment]
Need to leave open to address this in v3.0, as this can be easily handled within the newest function Invoke-ScriptBlockHandler once it is merged into the main Dev3.0 branch.

**Comment by dpaulson45** [comment]
Done within Dev3.0.0 in this commit d990ddf09c2507f21f122b24da894d3d627497be
